File: src/main/java/org/trackedout/citadel/commands/PartyCommand.java
package org.trackedout.citadel.commands;

import org.bukkit.entity.Player;
import org.trackedout.citadel.Citadel;
import org.trackedout.citadel.classes.Party;

import co.aikar.commands.BaseCommand;
import co.aikar.commands.annotation.Default;
import co.aikar.commands.annotation.Dependency;
import co.aikar.commands.annotation.Description;
import co.aikar.commands.annotation.Subcommand;

public class PartyCommand extends BaseCommand {
    @Dependency
    private Citadel plugin;

    @Default
    @Subcommand("info")
    @Description("Get info about your party")
    public void onPartyInfo(Player player) {

        Party party = this.plugin.parties.stream().filter(p -> p.members.contains(player)).findFirst().orElse(null);
        if (party == null) {
            player.sendMessage("You are not in a party.");
        } else if (party.leader == player) {
            player.sendMessage("You are the leader of a party with " + party.members.size() + " members.");
        } else {
            player.sendMessage("You are a member of a party with " + party.members.size() + " members.");
        }
    }

}

File: src/main/kotlin/org/trackedout/citadel/commands/PartyCommand.kt
package org.trackedout.citadel.commands

import co.aikar.commands.BaseCommand
import co.aikar.commands.annotation.*
import org.bukkit.entity.Player
import org.trackedout.citadel.Citadel

@CommandAlias("info|party")
class PartyCommand : BaseCommand() {
    @Dependency
    private lateinit var plugin: Citadel

    @Default
    @Subcommand("info")
    @Description("Get info about your party")
    fun onPartyInfo(player: Player) {
        plugin.parties.stream().filter { p -> p.members.contains(player) }.findFirst().ifPresent { party ->
            if (party.leader === player) {
                player.sendMessage("You are the leader of a party with ${party.members.size} members.")
            } else {
                player.sendMessage("You are a member of a party with ${party.members.size} members.")
            }
        }
    }
}